src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/UploadApprovedOrderService.java

```diff
@@ -20,49 +18,36 @@
 public class UploadApprovedOrderService {
     private final HearingOrderService hearingOrderService;
     private final ContestedOrderApprovedLetterService contestedOrderApprovedLetterService;
-    private final AdditionalHearingDocumentService additionalHearingDocumentService;
     private final ApprovedOrderNoticeOfHearingService approvedOrderNoticeOfHearingService;
 
     /**
      * Method for processing approved orders in a financial remedy case.
      * This method generates and stores the contested order approved letter, creates additional order documents,
      * appends the latest draft direction order, and updates the hearing order collection with approved hearing orders.
      *
-     * <p>Use {@link #processApprovedOrdersMh(FinremCaseDetails, FinremCaseDetails, String)} instead.</p>
+     * <p>Use {@link #processApprovedOrdersMh(FinremCaseDetails, String)} instead.</p>
      *
      * @param callbackRequest   the callback request containing case details
-     * @param errors            a list to collect error messages encountered during processing
      * @param authorisationToken the authorisation token for accessing secure resources
      * @deprecated This method is deprecated and should not be used. Scheduled for removal since 30/09/2025.
      */
     @Deprecated(forRemoval = true, since = "30/09/2025")
     @SuppressWarnings("squid:S1133") // Suppress SonarQube rule for deprecated code
     public void processApprovedOrders(FinremCallbackRequest callbackRequest,
-                                      List<String> errors,
                                       String authorisationToken) {
         FinremCaseDetails caseDetails = callbackRequest.getCaseDetails();
         contestedOrderApprovedLetterService.generateAndStoreContestedOrderApprovedLetter(caseDetails, authorisationToken);
-        try {
-            additionalHearingDocumentService.createAndStoreAdditionalHearingDocumentsFromApprovedOrder(authorisationToken, caseDetails);
-        } catch (CourtDetailsParseException e) {
-            log.error(e.getMessage());
-            errors.add(e.getMessage());
-        }
 
+        processCaseworkerUploadedApprovedOrders(caseDetails.getData(), authorisationToken);
+
+        // TODO Looks like the following logic is not needed. Remove it later.
         hearingOrderService.appendLatestDraftDirectionOrderToJudgesAmendedDirectionOrders(caseDetails);
```

Comment:
Consider removing before this PR as it's own change, or as part of this PR?

Reply:
You've another todo below.  Will stop raising, you can address if appropriate 👍

Reply:
It's the legacy logic that appends the document to `judgesAmendedOrderCollection`. However, I cannot see this field is being shown in our tabs.  Perhaps, we need another ticket to check whether this logic should be removed.

Reply:
The only usage is that the document will appear in CFV under the category "System Duplicated.
https://github.com/hmcts/finrem-case-orchestration-service/blob/ee0f58d3a029950afd96929d33f1fd55ca1e1452/src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/documentcatergory/UploadedDraftOrderCategoriser.java#L22

@ptrelease